apps/src/blockly/utils.ts

```diff
@@ -35,7 +35,7 @@ export const TRUTHY_DEFAULT = (attributeValue: xmlAttribute) =>
 export function readBooleanAttribute(
   xmlElement: Element,
   attribute: string,
-  callback = FALSEY_DEFAULT
+  callback: (attributeValue: xmlAttribute) => boolean = FALSEY_DEFAULT
```

Comment:
TS 5.5.3 got too smart and was auto-typing this as:
`(attributeValue: xmlAttribute) => attributeValue is "true"`

Note the `is "true"` which is overly specific, and caused cases where TRUTHY_DEFAULT was passed in to type error. Explicitly defining callback to widen its type's return signature fixes the issue.